### ISSUE TITLE
Broken wufoo flow

### ISSUE THREAD

**Issue by oliverbarnes** [BlueRidgeLabs/kimball#20]
The new registration flow added new validations for Person and broke the Wufoo registration flow. For now I've disabled the respective broken tests on de87068c0f70254926cbfcbd67e19cf7a2dec0e8 to keep travis happy.

What's the plan with the current Wufoo flow? Will it be replaced by the new flow, in which case we can simply remove the related tests together with the functionality, or will it continue in parallel and so we'll have to adapt the current functionality to the new validations?

**Comment by shua123** [comment]
Is this merged into your Development branch right now? What broke in the Wufoo registration flow?

**Comment by cromulus** [comment]
hey @shua123, right now, the new person flow we are using doesn't use Wufoo at all.

Ideally, we will eventually pull out those bits from the new person flow all together.
